cpu/native/syscalls.c

```diff
@@ -145,7 +145,7 @@ void free(void *ptr)
     _native_syscall_leave();
 }
 
-int _native_in_calloc;
+int _native_in_calloc = 0;
```

Comment:
I just wanted to explicate it because I wasn't immediately sure when rereading it while fixing the error.